about.html

```diff
@@ -0,0 +1,184 @@
+<!DOCTYPE html>
+<html lang="en">
+
```

Comment:
- [ ] Nice job for adding the _About_ page. However please provide a link on the _About_ page that will send visitors back to the _Home_ page when clicked. You can change the `about menu link` on your _About_ page to a `home link` within the menu.

Reply:
Thank you I've resolve this.